### PR TITLE
WIP: Output API logs

### PR DESCRIPTION
**Pull request by oomichi**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Enabe API logs.
Investigating which log can be used for verifying stable APIs only are called on conformance tests.

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: cluster/images/conformance/conformance-e2e.yaml [hunk 1]
```diff
@@ -67,7 +67,7 @@ spec:
     - name: E2E_PARALLEL
       value: "false"
     - name: E2E_VERBOSITY
-      value: "4"
+      value: "8"
```
Comment (timothysc) [inline cluster/images/conformance/conformance-e2e.yaml]: Again, I'm not a fan of doing this by default, it's going to be super noisy.
/hold 
/assign @spiffxp @timothysc

**Comment by spiffxp** [comment]
@oomichi I feel like we've said a number of times that we're not interested in bumping verbosity client-side, and that audit logs are available server side for this sort of data.  Maybe it's time to have a chat in slack or zoom to figure out what's going on?

**Comment by oomichi** [comment]
@spiffxp Thanks for your comment. It is nice to take conversation about that at this time.
(Sorry for my late response due to my early summer vacation last week)

**Comment by spiffxp** [comment]
/close
Please come discuss this in #k8s-conformance or the next conformance office hours and we can decide whether to /reopen this; we should be getting this from audit logs server-side

**Pull request closed without merging**